docs/cpp/faq-cpp.md

```diff
@@ -9,11 +9,11 @@ MetaDescription: Frequently asked questions about the C/C++ extension in Visual
 ---
 # Frequently asked questions
 
-- [Why are my files corrupted on format?](#why-are-my-files-corrupted-on-format)
 - [How do I get IntelliSense to work correctly?](#how-do-i-get-intellisense-to-work-correctly)
 - [Why do I see red squiggles under Standard Library types?](#why-do-i-see-red-squiggles-under-standard-library-types)
 - [How do I get the new IntelliSense to work with MinGW on Windows?](#how-do-i-get-the-new-intellisense-to-work-with-mingw-on-windows)
 - [How do I get the new IntelliSense to work with the Windows Subsystem for Linux?](#how-do-i-get-the-new-intellisense-to-work-with-the-windows-subsystem-for-linux)
+- [Why are my files corrupted on format?](#why-are-my-files-corrupted-on-format)
```

Comment:
Why did you decide to move this? I am going to see which ones of these links are clicked the most, so we could sort it most to least clicked, but I if you have another specific reason let me know.

Reply:
@AlexandraKemperMS I moved it because it was strange to be the 1st question since it's actually not a common issue that people frequently ask, i.e. the IntelliSense issues are more frequently asked.

Reply:
That sounds good - my instinct is also that people ask around IS etc. more often. We can leave this for now and if I find anything in the data (migth take a bit to find) we can potentially switch the order in a future PR.